lib/proactive_support/mgmt/flags.rb

```diff
@@ -40,7 +40,11 @@ def to_digest(source, identifier, filter)
         end
 
         def clear_matching(customer_id, conditions)
-          ::ProactiveSupport::Flag.where(customer_id: customer_id, is_active: true).update_all({is_active: false}, conditions)
+          if ActiveRecord::VERSION::MAJOR == 3
+            ::ProactiveSupport::Flag.where(customer_id: customer_id, is_active: true).update_all({is_active: false}, conditions)
+          else
```

Comment:
Should this else explicitly call out version 4?

Reply:
This is valid syntax from 4 to at least 6, so I think it's better like this for the future